core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/spreadsheet/SampleSpreadSheets.java

```diff
@@ -153,8 +140,81 @@ public enum SampleSpreadSheets implements Spreadsheet<Sample> {
       Column.forString("QC Status Note", true, detailedSample(DetailedSample.class, DetailedSample::getDetailedQcStatusNote, "")), //
       Column.forString("Notes",
           sam -> sam.getNotes() == null ? "" : sam.getNotes().stream().map(Note::getText).collect(Collectors.joining("; ")))
+      ), //
+  RUNNING_SHEET("Running Sheet", //
+      Arrays.asList(SampleStock.CATEGORY_NAME, SampleAliquot.CATEGORY_NAME), //
+      Column.forString("Alias", Sample::getAlias), //
+      Column.forString("Group ID", effectiveGroupIdProperty(GroupIdentifiable::getGroupId)),
+      Column.forString("", sam -> ""), // intentional blank column
```

Comment:
Why is this one 'sam -> ""' instead of blankColumn()?

Reply:
Fixed, thanks!